pages/shared/data/roadshow.yaml

```diff
@@ -1,12 +1,36 @@
-# nextstop:
-  # - title: California, US
-  #   coords: 9x9
-  #   location: Menlo Park
-  #   time: 09:00–18:00
-  #   day: 4
-  #   month: sep
-  #   year: 1998
-  #   link: https://google.com/
+nextstop:
+  - title: tel aviv
+    coords: 64x21
```

Comment:
I'd suggest changing Tel Aviv coords to 66x22 and Istanbul coords to 64x19 to fix the overlapping issue. This would fix the overlapping issue.
![fix-coords](https://user-images.githubusercontent.com/22053023/60598631-4f7ce900-9dad-11e9-8859-c7dbc70eb432.png)